tests/tests/eth_provider.rs

```diff
@@ -20,6 +20,7 @@ use reth_primitives::{
 };
```

Comment:
Is there no way to fix this from within the mongo fuzzer structure? For example by adding a `current_block` field to the fuzzer and using this to set the block number of logs?

Reply:
Just changed that directly inside the Mongo Fuzzer. Let us check with the CI if it doesn't break anything else in the tests.